docs/examples/yang19.ipynb

```diff
@@ -0,0 +1,573 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy\n",
+    "import numpy as np\n",
+    "import neurogym as ngym\n",
+    "from neurogym.wrappers import ScheduleEnvs\n",
+    "from neurogym.utils.scheduler import RandomSchedule\n",
+    "from neurogym.utils import plot_env\n",
+    "import gymnasium as gym\n",
+    "import os\n",
+    "import sys\n",
+    "from neurogym.wrappers.monitor import Monitor\n",
+    "import torch.nn as nn\n",
+    "import torch\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1.2 Environment Setup\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Environment parameters\n",
+    "EVAL_TRIALS = 1000\n",
+    "dt = 100\n",
+    "dim_ring = 2  # Number of choices in the ring representation\n",
+    "rewards = {\n",
+    "    \"abort\": -0.1,\n",
+    "    \"correct\": +1.0,\n",
+    "    \"fail\": 0.0\n",
+    "}\n",
+    "sigma = 1.0 # Standard deviation of the Gaussian noise in the ring representation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### dm1\n",
+    "timing = {\n",
+    "    \"fixation\": (\"uniform\", (200, 500)),\n",
+    "    \"stimulus\": (\"choice\", [200, 400, 600]),\n",
+    "    \"decision\": 200,\n",
+    "}\n",
+    "kwargs = {\n",
+    "    \"dt\": dt,\n",
+    "    \"dim_ring\": dim_ring,\n",
+    "    \"rewards\": rewards,\n",
+    "    \"timing\": timing,\n",
+    "    \"sigma\": sigma,\n",
+    "}\n",
+    "task = 'dm1'\n",
+    "task = f\"yang19.{task}-v0\"\n",
+    "env_dm = gym.make(task, **kwargs)\n",
+    "\n",
+    "# Print environment specifications\n",
+    "print(\"Trial timing (in milliseconds):\")\n",
+    "print(env_dm.timing)\n",
+    "\n",
+    "print(\"\\nObservation space structure:\")\n",
+    "print(env_dm.observation_space)\n",
+    "\n",
+    "print(\"\\nAction space structure:\")\n",
+    "print(env_dm.action_space)\n",
+    "print(\"Action mapping:\")\n",
+    "print(env_dm.action_space.name)\n",
+    "\n",
+    "obs, info = env_dm.reset()\n",
+    "\n",
+    "stim1 = [f'Stim {i}, Mod 1' for i in range(1, kwargs['dim_ring'] + 1)]\n",
+    "stim2 = [f'Stim {i}, Mod 2' for i in range(1, kwargs['dim_ring'] + 1)]\n",
+    "\n",
+    "# Visualize example trials\n",
+    "fig = plot_env(\n",
+    "    env_dm,\n",
+    "    name='DM1',\n",
+    "    ob_traces=['Fixation'] + stim1 + stim2,\n",
+    "    num_trials=5,\n",
+    "    plot_performance=True,\n",
+    "    fig_kwargs={'figsize': (9, 5)},\n",
+    ")\n",
+    "\n",
+    "for text_obj in fig.findobj(match=plt.Text):\n",
+    "    text_obj.set_fontsize(11)\n",
+    "fig.savefig(f\"untrained_{task}.pdf\", dpi=300, bbox_inches='tight')"
```

Comment:
please remove the `savefig`s (there a a couple more below).
or comment them out so that users have the export code as a reference if they want to export.